build/quickstarts-showcase/pom.xml

```diff
@@ -167,6 +167,65 @@
         <quarkus.package.type>native</quarkus.package.type>
       </properties>
     </profile>
+    <profile>
+      <id>container</id>
+      <activation>
+        <property>
+          <name>container</name>
+        </property>
+      </activation>
+      <properties>
+        <container.runtime>podman</container.runtime>
+      </properties>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-assembly-plugin</artifactId>
+            <executions>
+              <execution>
+                <phase>package</phase>
+                <goals>
+                  <goal>single</goal>
+                </goals>
+              </execution>
+            </executions>
+            <configuration>
+              <descriptors>
+                <descriptor>src/main/assembly/assembly-timefold-quickstarts.xml</descriptor>
+              </descriptors>
+              <finalName>container-content</finalName>
+              <appendAssemblyId>false</appendAssemblyId>
+            </configuration>
+          </plugin>
+          <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>exec-maven-plugin</artifactId>
```

Comment:
Is there a specialized docker maven plugin (the jib plugin maybe?) who can reduce the boilerplate in this exec approach?

Reply:
Happy to push this to a later issue (let's create on then)

Reply:
There is no need for any maven plug-in nor docker file. Just use jib quarkus extension and that’s it. Same as we do for nga and now also for employee scheduling in platform

Reply:
The jib quarkus extension is a great thing, if the goal is to containerize a single quarkus app. Unfortunately, that's not what's happening here: there are 9 different applications in the container and one call the others.

To have all the content inside, we have to:

1. build the showcase app
2. grab the individual quickstarts and pack it with the showcase; that's what the assembly plugin does
3. put it all in a container image

The step 1. is when the maven quarkus plugin runs, including the jib extension.

After experimenting with two runs of the quarkus plugin, I also tried the jib maven plugin. Unfortunately, it does not allow just building the image locally; it always tries to push to some docker registry. What means you either have a local registry running or use the github registry directly. In my opinion, that is worse than having either `podman` or `docker` installed. Btw., you can switch the container runtime with a maven property.